libopenage/renderer/renderer.h

```diff
@@ -60,11 +60,19 @@ struct Renderable {
 };
 
 /// A render pass is a series of draw calls represented by renderables that output into the given render target.
-struct RenderPass {
-	/// The renderables to parse and possibly execute.
-	std::vector<Renderable> renderables;
+class RenderPass {
+protected:
+    RenderPass(std::vector<Renderable>, RenderTarget const*);
+    /// The renderables to parse and possibly execute.
+    std::vector<Renderable> renderables;
+
+public:
+    virtual ~RenderPass() = default;
+	void set_target(RenderTarget const*);
+	RenderTarget const* get_target() const;
+private:
 	/// The render target to write into.
-	RenderTarget const *target;
+	RenderTarget const* target;
```

Comment:
our codebase has the convention to align `*` and `&` to the right (because semantically the `*` belongs to the variable, and not the type)